src/data/local_storage.js

```diff
@@ -71,6 +71,10 @@ p5.prototype.storeItem = (key, value) => {
     case 'object':
       if (value instanceof p5.Color) {
         type = 'p5.Color';
+      } else if (value instanceof p5.Vector) {
+        type = 'p5.Vector';
+        let coord = [value.x, value.y, value.z];
```

Comment:
```suggestion
        const coord = [value.x, value.y, value.z];
```

small request, to try to use `const` whenever something doesn't need to be reassigned. Otherwise looks great!